### DISCUSSION TITLE
Can't upload to Repository.  remote: Repository not found.

### DISCUSSION THREAD

**Discussion by tonysterrett** [community/community#23007] [New to GitHub]
<p>Hello:</p>
<p>I have seen similar problem to mine, however, those solutions are not working. </p>
<p>I did the following : </p>
<pre><code>git remote add origin https://github.com/ManufADE/rpio4.gitgit push -u origin master
</code></pre>
<p>On a repository which had already been set up. I get: </p>
<p>remote: Repository not found.<br>
fatal: repository ‘<a href="https://github.com/ManufADE/rpio4.git/" rel="nofollow noopener">https://github.com/ManufADE/rpio4.git/</a>’ not found</p>
<p>Now, <a href="https://github.com/ManufADE/rpio4" rel="nofollow noopener">https://github.com/ManufADE/rpio4</a> exists! And I copied the above git commands directly from GitHub’s.  page (as is recomended) . </p>
<p>Does anyone have a solution or some suggested steps for debug. </p>

**Comment by tonysterrett** [comment]
<p>BTW, if I do:</p>
<pre><code>git remote add origin https://github.com/ManufADE/rpio4.git
</code></pre>
<p>Again i get:</p>
<p>fatal: remote origin already exists. </p>

**Comment by tonysterrett** [comment, marked as answer]
<p>Resolved. Couldn’t figure this one out. I used the github app and it wroked</p>

**Comment by Uniqueorp** [comment]
But what you did with the app I am facing the same problem, just the difference is that I created my account about 3 days ago with a  custom business email , and a another difference in my case  is that my public repo's such as this one https://github.com/Uniqueorp/portfoliov1  show 404 page not found.
